tests/installation/partitioning/encrypt_lvm.pm

```diff
@@ -18,7 +18,7 @@ use testapi;
 
 sub run {
     my $partitioner = $testapi::distri->get_partitioner();
-    $partitioner->edit_proposal(is_lvm => 1, is_encrypted => 1);
+    $partitioner->edit_proposal(multiple_disks => get_var('IBFT'), is_lvm => 1, is_encrypted => 1);
```

Comment:
Do you think it's worth the effort to create `has_multiple_disks` helper function? We could also add check for `NUMDISKS` variable.

Reply:
Makes sense, more clean, I fixed, is what you have in mind? I'm re-running [this one](https://openqa.opensuse.org/tests/1322698) just in case.